README.md

```diff
@@ -1,2 +1,99 @@
-# tw-academy-s9
-Activity for session 9
+# **Sample Style Guide**            
+### Technical Writing Academy - May 2021
+
+### **Table of contents**
+
+#### [Bold](Bold)
```

Comment:
I think I found the problem with anchors. You are supposed to include in the parentheses the same number of "#" from the header. For example, [Bold](###Bold)